app/server/services/setupSecrets.ts

```diff
@@ -0,0 +1,22 @@
+import * as config from 'config';
+import * as propertiesVolume from '@hmcts/properties-volume';
+import { get, set } from 'lodash';
+
+propertiesVolume.addTo(config);
+
+function setSecret (secretPath: string, configPath: string) {
+  // Only overwrite the value if the secretPath is defined
+  if (config.has(secretPath)) {
+    set(config, configPath, get(config, secretPath));
+  }
+}
+
+export function setupKeyVaultSecrets () {
```

Comment:
```suggestion
export function setupKeyVaultSecrets (): void {
```

could also make it the default function as I don't expect you to need to export anything else from here,

```suggestion
export default function setupKeyVaultSecrets () {
```